static/locales/en-US.yaml

```diff
@@ -322,6 +322,7 @@ Settings:
     Hide Active Subscriptions: Hide Active Subscriptions
     Hide Video Description: Hide Video Description
     Hide Comments: Hide Comments
+    Show distraction free titles: Show distraction free titles
```

Comment:
Could you please choose a descriptive name for the setting, one that explains what it does. I assume you chose this name, as you want to expand on the functionality of this setting later on, however it's better to use a descriptive name until then. Keep in mind that most users won't dig into the source code.